lms/djangoapps/django_comment_client/base/views.py

```diff
@@ -116,6 +149,28 @@ def create_thread(request, course_id, commentable_id):
     if 'pinned' not in thread.attributes:
         thread['pinned'] = False
 
+    follow = post.get('auto_subscribe', 'false').lower() == 'true'
+
+    if follow:
+        user = cc.User.from_django_user(request.user)
+        user.follow(thread)
+
+    event_data = {
+        'title': thread.title,
+        'commentable_id': commentable_id,
+        'options': {'followed': follow},
+        'anonymous': anonymous,
+        'thread_type': thread.thread_type,
+        'group_id': group_id,
+        'anonymous_to_peers': anonymous_to_peers,
+        # There is a stated desire for an 'origin' property that will state
+        # whether this thread was created via courseware or the forum.
```

Comment:
Yup, I remember, and still think that's the right call. We are pushing through an entirely separate PR that ensures we track the referrer on all events, see #6867